### DISCUSSION TITLE
should positive ATK tests (probable cases) be added to daily new cases?

### DISCUSSION THREAD

**Discussion by chrisadas** [djay/covidthailand#119] [Q&A] [answered]
Should they be added? Or perhaps a new column with new cases combining official number and ATK test?

**Comment by djay** [comment, marked as answer]
No they should not be added because of double counting issues. It's not
possible to tell how many atk become confirmed cases.
It is already recorded In a seperate column and shown on testing plot.
I did originally show this number of the cases by type plot but that has
the same problem that it is showing a misleading number because there could
be double counting.
It's also unclear that atk isn't just finding more of the undiscovered
infections that lack of pcr testing could find so adding it now could
create a misleading trend

**Comment by djay** [comment]
This is the plot ATK is current shown in.
I'm open to other ideas to show it more prominently on the site but I think adding to cases is going to be misleading.
![image](https://github.com/djay/covidthailand/wiki/cases_3.png)

**Comment by djay** [comment]
Still can't really use it but there is a interesting plot that appears in at least one briefing ```inputs/briefings/221064.pdf ```
I'm assuming this is only proactive ATK tests otherwise how would they know the negatives? Translates as 
```
Number and proportion of daily and cumulative domestic ATK positives 20 Aug. - 22 Oct. '64
```
 but not sure what domestic that means? 

<img width="1217" alt="Screen Shot 2021-10-25 at 1 04 52 pm" src="https://user-images.githubusercontent.com/41700/138642712-e25f7193-604a-47d4-ab3a-e6f2832dbdd6.png">
